src/utils/chat-gpt.ts

```diff
@@ -18,13 +18,22 @@ async function askGPTChat(messages, debug) {
             logger.info('-----Question-----');
             logger.info(stringifyJSON(messages));
             logger.info('');
+            logger.info('-----OpenAI Config-----');
+            logger.info(
+                stringifyJSON({
+                    model: process.env.GPT_MODEL ?? 'gpt-4',
```

Comment:
Make sure to add an entry in the env sample file. Also change the name to `OPENAI_GPT_MODEL`.

Reply:
> Make sure to add an entry in the env sample file. Also change the name to `OPENAI_GPT_MODEL`.

done